src/api/json/catalog.json

```diff
@@ -993,6 +993,44 @@
       "fileMatch": ["dbt_project.yml"],
       "url": "https://raw.githubusercontent.com/dbt-labs/dbt-jsonschema/main/schemas/latest/dbt_project-latest.json"
     },
+    {
+      "name": "dbt Packages",
+      "description": "dbt packages configurations",
+      "fileMatch": ["**/packages.yml"],
+      "url": "https://raw.githubusercontent.com/dbt-labs/dbt-jsonschema/main/schemas/latest/packages-latest.json"
+    },
+    {
+      "name": "dbt Selectors",
+      "description": "dbt selectors configurations",
+      "fileMatch": ["**/selectors.yml"],
+      "url": "https://raw.githubusercontent.com/dbt-labs/dbt-jsonschema/main/schemas/latest/selectors-latest.json"
+    },
+    {
+      "name": "dbt YAML files",
+      "description": "dbt YAML files configurations",
+      "fileMatch": [
+        "**/macros/*.yml",
+        "**/macros/**/*.yml",
+        "**/macros/*.yaml",
+        "**/macros/**/*.yaml",
+
+        "**/models/*.yml",
+        "**/models/**/*.yml",
+        "**/models/*.yaml",
+        "**/models/**/*.yaml",
+
+        "**/seeds/*.yml",
+        "**/seeds/**/*.yml",
+        "**/seeds/*.yaml",
+        "**/seeds/**/*.yaml",
+
+        "**/snapshots/*.yml",
+        "**/snapshots/**/*.yml",
+        "**/snapshots/*.yaml",
+        "**/snapshots/**/*.yaml"
+      ],
```

Comment:
We can technically configure the directories instead of the default directories respectively. If we want the rules more flexible, we need to parse all YAML files. But, I'm not sure we can apply file match rules like `**/*.yml`, `**/**/*.yml`.